dtale/utils.py

```diff
@@ -282,18 +282,14 @@ def json_date(x, fmt="%Y-%m-%d %H:%M:%S", nan_display="", **kwargs):
 def json_timestamp(x, nan_display="", **kwargs):
     """
     Convert value to timestamp (milliseconds) to be used within JSON output
-
     :param x: value to be converted to milliseconds
     :param nan_display: if `x` is :attr:`numpy:numpy.nan` then return this value
     :return: millisecond value
     :rtype: bigint
     """
     try:
         output = pd.Timestamp(x) if isinstance(x, np.datetime64) else x
-        output = int(
-            (time.mktime(output.timetuple()) + (old_div(output.microsecond, 1000000.0)))
-            * 1000
-        )
+        output = int(output.timestamp() * 1000)
```

Comment:
I'd revert this as we discussed.  Path of least resistance.  This will be a good fight for the next PR 😉